spec/rubocop/cop/lint/inline_access_modifier_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Lint::InlineAccessModifier do
+  subject(:cop) { described_class.new }
+
+  %w[private protected public].each do |access_modifier|
```

Comment:
Please prefer [DAMP](http://vgaltes.com/code/testing/testing-damp-vs-dry/) tests using the `#expect_offense` and `#expect_no_offenses` matchers. A few extra lines is worth it for the less obscure test cases, and having to reverse engineer a failing test case. 🙂

Reply:
👌Totally agree—DAMP FTW. Sorry, there are so many different ways tests are written in this codebase I couldn't tell what the preferred way is. I will update them!